### DISCUSSION TITLE
Move relation box position on Form

### DISCUSSION THREAD

**Discussion by kinzinho666** [filamentphp/filament#12390] [Help] [answered]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

Is possible change position for my relation?

`public static function getRelations(): array
    {
        return [
            DocumentoCompsRelationManager::class,
        ];
    }`

**Problem:** my last item on Form are multiple pictures (100+), so  my relation "fades away" in final on page, i need something like this:

`
Forms\Components\RichEditor::make('texto')
                    ->required()
                    ->columnSpanFull(),

                 Forms\Components\Section::make(DocumentoCompsRelationManager::class)

                Forms\Components\Section::make('Imagem chamada')
                    ->hiddenLabel()
                    ->collapsible()
                    ->schema([
                        Forms\Components\FileUpload::make('image_chamada')
                        ->hiddenLabel()
                        ->minSize(20)
                        ->multiple
                        ->maxSize(6000)
                        ->hint('Tamanho máximo permitido: 5mb.')
                        ->acceptedFileTypes(['image/png', 'image/jpg', 'image/jpeg'])
                        ->directory('competicoes/chamada')
                    ]),`

How i can do?

**Comment by dissto** [comment, marked as answer]
Afaik you can't. But what you can try is:

https://filamentphp.com/docs/3.x/panels/resources/relation-managers#combining-the-relation-manager-tabs-with-the-form
